docs/BestPracticesObjC.md

```diff
@@ -70,7 +70,7 @@ namespace Xamarin.Xml.Configuration {
 would be used like:
 
 ```
-id reader = [[Xamarin_Xml_Configuration_.Reader alloc] init];
+id reader = [[Xamarin_Xml_Configuration.Reader alloc] init];
```

Comment:
it should be `id reader = [[Xamarin_Xml_Configuration_Reader alloc] init];` (no dot, but an underscore)